docs/book/src/cronjob-tutorial/testdata/project/go.mod

```diff
@@ -5,7 +5,6 @@ go 1.12
 require (
 	github.com/go-logr/logr v0.1.0
 	github.com/robfig/cron v1.2.0
-	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 // indirect
```

Comment:
Non-blocking: Can I ask why are you removing this? I already asked it but I didn't understand your point. I just ask this because my IDE is automatically updating two go-mod files already adding this line because it does think it is needed. I would like to understand what is the reason behind this removal.